libosmscout/include/osmscout/util/TagErrorReporter.h

```diff
@@ -0,0 +1,50 @@
+/*
+  This source is part of the libosmscout library
+  Copyright (C) 2017 Lukas Karas
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307  USA
+*/
+
+#ifndef TAGERRORREPORTER_H
+#define TAGERRORREPORTER_H
```

Comment:
A header should make sure that all included, it requires are included directly. This will avoid spurious errors in cases where included are added/removed.